website_sale_cart_preview/README.md

```diff
@@ -0,0 +1,29 @@
+Cart Preview
+============
+
+This module adds a preview of your cart on mouseover.
+
+Credits
+=======
+
+This module is a backport of the v9 implementation of this feature.
+
+Contributors
+------------
+
+* Leonardo Donelli <donelli@webmonks.it>: Initial author of the module
```

Comment:
Here it should be original extraction and adaptation, but we don't usually put this info: commits tell us this.